packages/js/product-editor/src/blocks/product-fields/product-details-section-description/editor.scss

```diff
@@ -0,0 +1,52 @@
+.wp-block-woocommerce-product-details-section-description {
+	margin-top: $grid-unit;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	flex-wrap: wrap;
+	gap: $grid-unit-05;
+
+	p {
+		margin: 0;
+	}
+
+	&__dropdown .components-button.components-menu-item__button {
+		border-radius: 2px;
+		padding: $grid-unit $grid-unit + 2;
+		padding-inline-start: $grid-unit-30 + $grid-unit-05;
+
+		&:hover {
+			background-color: $gray-100;
+			color: inherit;
+		}
+
+		&.has-icon {
+			gap: $grid-unit-05;
+			padding-inline-start: 0;
+
+			svg {
+				flex-shrink: 0;
+				align-self: flex-start;
+			}
+		}
+
+		&--selected {
+			svg,
+			.components-menu-item__item {
+				color: var(--wp-admin-theme-color);
+			}
+		}
+	}
+
+	&__modal {
+		max-width: 650px;
```

Comment:
Is there a variable that was can use for the max width of a modal in our system rather than hard-coding this?